prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -26,6 +28,7 @@ import java.time.ZoneId
 import java.time.format.DateTimeFormatter
 import java.util.Properties
 import org.apache.logging.log4j.kotlin.Logging
+import java.text.SimpleDateFormat
 
```

Comment:
I used it for convenience as the HAPI uses the Date class and it is just quicker, but I can change it.  There are other uses of SimpleDateFormat in the code base that would have to be migrated at some point.

> I'm wondering why the old Java 7 class is being used instead of the more modern Java 8 DateTimeFormatter.

Reply:
Changed